### PR TITLE
Warn if hyperkit version is old

### PR DESCRIPTION
**Pull request by josedonizetti**
Fix for https://github.com/kubernetes/minikube/issues/4658

### CODE REVIEW

File: cmd/minikube/cmd/start.go [hunk 1]
```diff
@@ -856,39 +856,54 @@ func saveConfig(clusterConfig cfg.Config) error {
 }
 
 func validateDriverVersion(vmDriver string) {
```
Comment (medyagh) [inline cmd/minikube/cmd/start.go]: is there a reason we don't do this validation before download ISO ?

**Comment by sharifelgamal** [comment]
@minikube-bot OK to test

**Comment by josedonizetti** [comment]
@minikube-bot OK to test

File: cmd/minikube/cmd/start.go [hunk 2]
```diff
@@ -856,39 +855,54 @@ func saveConfig(clusterConfig cfg.Config) error {
...
-			return
-		}
+	var (
+		driverExecutable    string
+		driverDocumentation string
+	)
+
+	switch vmDriver {
+	case constants.DriverKvm2:
+		driverExecutable = "docker-machine-driver-kvm2"
```
Comment (tstromberg) [inline cmd/minikube/cmd/start.go]: Please move these constants somewhere else - preferably as close to the driver as possible. I'm afraid we may  silently break this in the future, should we rename from kvm to kvm2, for instance.

Comment (josedonizetti) [reply cmd/minikube/cmd/start.go]: Let me know if the last change is enough, or if you prefer to actually create a constant inside the `pkg/drivers/kvm|hyperkit`.

**Comment by tstromberg** [comment]
LGTM, but needs rebase.

**Comment by RA489** [comment]
@josedonizetti please resolve the conflicts rest LGTM.

**Pull request merged**